core/controllers/classroom_test.py

```diff
@@ -381,6 +519,80 @@ def test_update_classroom_data(self) -> None:
 
         self.logout()
 
+    def test_update_classroom_thumbnail_data_only_should_not_return_error(
```

Comment:
Should you also be checking that the update actually worked (i.e. the data for both modified and unmodified fields are as expected at the end of the operation)?

Reply:
Got it. I've added checks at the end of this test and the next one to verify that the classroom data is updated and the image is uploaded correctly.